SPECS-SIGNED/kernel-uki-signed/kernel-uki-signed.spec

```diff
@@ -0,0 +1,70 @@
+%global debug_package %{nil}
```

Comment:
Just curious, what was the reasoning behind keeping kernel-uki under kernel but making kernel-uki-signed its own folder?

Reply:
I think we just forgot to extract the spec out to a new folder 😓 but ideally we still want to have kernel-uki as part of the kernel spec